server/models/SentenceTypes.ts

```diff
@@ -59,4 +87,21 @@ export default class SentenceTypes {
   public static isSentenceEdsRecall(sentence: PrisonApiOffenderSentenceAndOffences): boolean {
     return this.edsRecallSentenceTypes.includes(sentence.sentenceCalculationType)
   }
+
+  public static isSentenceStandardRecall(sentence: PrisonApiOffenderSentenceAndOffences): boolean {
+    return this.standardRecallSentenceTypes.includes(sentence.sentenceCalculationType)
+  }
+
+  public static isActiveRecall(sentence: PrisonApiOffenderSentenceAndOffences): boolean {
+    return (
+      this.isSentenceEdsRecall(sentence) ||
+      this.isSentenceSopcRecall(sentence) ||
+      this.isSentenceFixedTermRecall(sentence) ||
+      this.isSentenceStandardRecall(sentence)
+    )
+  }
+
+  public static isSentenceErsedElligible(sentence: PrisonApiOffenderSentenceAndOffences): boolean {
```

Comment:
I think this one could be done like above. 

```
!(
      this.isActiveRecall(sentence) || 
      this.isSentenceAfine(sentence) ||
      this.isSentenceDto(sentence)
      )
```

This would get rid of the duplicate sentence types in `ersedInelligibleSentenceTypes`. You'd need to create a new list for dto sentences

```
private static dtoSentenceTypes = [
    'DTO_ORA',
    'DTO'
]
```

Reply:
Done